src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -305,6 +335,11 @@ def update_existing_check_in(
         )
 
     if not valid_duration(updated_duration):
+        processing_errors.append(
+            ProcessingError(
+                ProcessingErrorType.CHECKIN_INVALID_DURATION, {"duration": updated_duration}
```

Comment:
Super nit: may as well add the magic trailing comma to get black to newline this like you did up here https://github.com/getsentry/sentry/pull/70145/files#diff-107af27baa83506a99019b39a35654dcb0e19c2e5a3bef9181a0de9185c62560R281-R282